src/main/scala/com/atomist/rug/runtime/js/JavaScriptInvokingProjectGenerator.scala

```diff
@@ -31,7 +31,7 @@ class JavaScriptInvokingProjectGenerator(
     val tr = time {
       val pmv = new ProjectMutableView(rugAs, startProject, atomistConfig = DefaultAtomistConfig, context)
 
-      val result = invokeMemberWithParameters("populate", wrapProject(pmv), poa)
+      val result = invokeMemberWithParameters("populate", wrapProject(pmv), pmv.name, poa)
```

Comment:
That will give you the name of the generator backing artefact source not the parameter named project_name.